src/app/core/cache/response-cache.models.ts

```diff
@@ -140,7 +140,7 @@ export class ErrorResponse extends RestResponse {
 
   constructor(error: RequestError) {
     super(false, error.statusText);
-    console.error(error);
+    // console.error(error);
```

Comment:
Unclear why this is commented out. Should it be removed or uncommented?

Reply:
I think this accidentally got commented out when I was working on another branch for handling errors. I'll reverse this change (in this branch).